Samples/Samples/Behaviors/EventToCommandBehaviour.cs

```diff
@@ -0,0 +1,132 @@
+﻿using System;
```

Comment:
Please include a comment as to where this code came from

Reply:
Added code reference in latest commit see: https://github.com/builttoroam/Essentials/pull/17/commits/ea7116121923e124f1f671fbc1b589b13302158f

Reply:
Can you do the same for the BehaviorBase class

Reply:
Done